### ISSUE TITLE
Mixing Keras layers and tf.layers  in Estimator  ( or otherwise)

### ISSUE THREAD

**Issue by whatdhack** [tensorflow/tensorflow#19465]
I am trying to figure out a way to mix Kears   and tf layers. Would be great  to be able to do something like the following. 

  x = tf.keras.layers.Dense(64, activation='relu', name ='d1')(inputs)
  x = tf.keras.layers.Dense(64, activation='relu', name ='d2')(x)
  predictions = tf.layers.Dense (units=10, activation=tf.nn.softmax, name='d3')(x)

Have I written custom code : NO
OS Platform and Distribution: N/A
TensorFlow installed from : N/A
TensorFlow version: 1.8
Bazel version: N/A
CUDA/cuDNN version: N/A
GPU model and memory: N/A
Exact command to reproduce: N/A

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 19 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by whatdhack** [comment]
Yes,   as far as I can tell, it is still an issue.

**Comment by facaiy** [comment]
I think you'd better use tf.keras.layer only if possible, because tf.layers will be depreciated in the future.

**Comment by whatdhack** [comment]
Tfdb and tensorboard debugger, histogram, distribution etc support are not supported  yet for keras.layers.   Can someone throw any light  on  the plan for debugging features for keras.layers ?

**Comment by facaiy** [comment]
Could you explain in more details (say, give an example)? I'm not familiar with debugger, however it sounds unreasonable for me that tensorboard doesn't support tf.keras.layers.

**Comment by angersson** [comment]
Thanks for the help, @facaiy!

This sounds like a question for [Stack Overflow](http://stackoverflow.com/questions/tagged/tensorflow), or a feature request for https://github.com/tensorflow/tensorboard, so I'm closing this issue.
